src/py42/services/cases.py

```diff
@@ -0,0 +1,195 @@
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.settings import items_per_page
+
+
+class CasesService(BaseService):
+    """`Rest documentation <https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases>`__ ."""
```

Comment:
Not a big deal, but I don't think this service needs class/method documentation.

Reply:
Only the client and the file events service do show up in the rst file and are directly called by the user

Reply:
Removed link from `service/cases`